rewrite-javascript/src/main/javascript/execution.ts

```diff
@@ -0,0 +1,5 @@
+export type ExecutionContext = Map<string | symbol, any> & { __brand: "ExecutionContext" }
```

Comment:
👍🏼 big fan of [opaque types](https://github.com/zieka/bonsai-components/blob/main/packages/utility-types/src/opaque-types.ts#L4)